README.md

```diff
@@ -1,2 +1,17 @@
 # sourcerer-app
 Sourcerer app makes engineering profiles from their git repositories.
```

Comment:
We are experimenting with language. Can you please change it to:

Sourcerer hashes your git repos into intelligent engineering profiles.